src/main/kotlin/it/pagopa/helpdeskcommands/controllers/CommandsController.kt

```diff
@@ -20,18 +20,26 @@ class CommandsController(@Autowired private val commandsService: CommandsService
         refundRedirectRequestDto: Mono<RefundRedirectRequestDto>,
         exchange: ServerWebExchange?
     ): Mono<ResponseEntity<RefundRedirectResponseDto>> {
-        // mocked implementation
         return refundRedirectRequestDto.flatMap { requestDto ->
             logger.info(
-                "Received refund redirect request for transactionId: [{}]",
+                "Received refund redirect request for idPSPTransaction: [{}]",
```

Comment:
`transactionId` and `idPspTransaction` are two different things, even if `transactionId` is not used to invoke the refund to the psp it is important to leave it in the request to log it (in a later feature it will be checked if the two data are valid)

Reply:
updated logger with [4172332](https://github.com/pagopa/pagopa-ecommerce-helpdesk-commands-service/pull/39/commits/4172332c302f0c276c7a666a6d1953b7bb9cdda7)